client/my-sites/patterns/home/index.tsx

```diff
@@ -16,10 +18,28 @@ import ImgLayout from './images/layout.svg';
 import ImgPattern from './images/pattern.svg';
 import ImgResponsive from './images/responsive.svg';
 import ImgStyle from './images/style.svg';
-import type { PatternGalleryFC } from 'calypso/my-sites/patterns/types';
+import type { Pattern, PatternGalleryFC } from 'calypso/my-sites/patterns/types';
 
 import './style.scss';
 
+function filterPatternsByQuery( patterns: Pattern[], searchTerm: string ) {
+	if ( ! searchTerm ) {
+		return [];
+	}
+
+	return patterns.filter( ( pattern ) => {
+		const lowercaseSearch = searchTerm.toLowerCase();
```

Comment:
We should move that line outside the `filter` call 🙂.

Reply:
Oh, and also update its name to be properly camelCased and to end with `Term`:
```suggestion
		const lowerCaseSearchTerm = searchTerm.toLowerCase();
```